app/code/Magento/Catalog/Controller/Adminhtml/Product/SuggestAttributeSets.php

```diff
@@ -46,4 +46,15 @@ public function execute()
         );
         return $resultJson;
     }
+    
+    /**
+     * Check for is allowed
+     *
+     * @return boolean
+     */
+    protected function _isAllowed()
```

Comment:
You are right but why this logic isn't used nowhere in project?
For instance:
Magento_Catalog/Controller/Adminhtml/Category.php
Magento_Catalog/Controller/Adminhtml/Product.php
Magento_Cms/Controller/Adminhtml/Block.php
Magento_Sales/Controller/Adminhtml/Order.php
etc.